src/components/TheNavigation.vue

```diff
@@ -59,7 +59,9 @@ export default {
         breed.isActive = false;
       });
       this.index--;
+
       this.breeds[this.lastState[this.index]].isActive = true;
+      this.$emit("countdown", this.index);
```

Comment:
a rabiš različna eventa za up in down eventa? ker načeloma kliče isto funkcijo, samo z različnim id-jem..